### PR TITLE
Update components.md (providers)

### PR DESCRIPTION
**Pull request by MickL**
Added related providers topics that cant be found otherwise without reading the whole documentation.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Other... Please describe: Updated documentation
```

### CODE REVIEW

File: content/components.md [hunk 1]
```diff
@@ -203,3 +203,11 @@ This is how our directory structure should look now:
...
 </div>
 </div>
+
+#### Get / Instantiate providers dynamically
+To get existing or instantiate providers dynamically you can use [Module reference](https://docs.nestjs.com/fundamentals/module-ref).
+
+#### Get providers inside bootstrap()
+To get providers within the `bootsrap()` function, e.g. for standalone applications without controllers, see [Standalone applications](https://docs.nestjs.com/standalone-applications)
+
+#### Get / Instantiate providers dynamically
```
Comment (jmcdo29) [inline content/components.md:213]: I think this shouldn't be here, as you already have the same section above.

File: content/components.md [hunk 2]
```diff
@@ -203,3 +203,11 @@ This is how our directory structure should look now:
 <div class="item">main.ts</div>
 </div>
 </div>
+
+#### Get / Instantiate providers dynamically
```
Comment (johnbiundo) [inline content/components.md:207]:
```suggestion
#### Manual instantiation

Thus far, we've discussed how Nest automatically handles most of the details of resolving dependencies.  In certain circumstances, you may need to step outside of the built-in Dependency Injection system and manually retrieve or instantiate providers.  We briefly discuss two such topics below.

 To get existing instances, or instantiate providers dynamically, you can use [Module reference](https://docs.nestjs.com/fundamentals/module-ref).
 
 To get providers within the `bootstrap()` function (for example for standalone applications without controllers, or to utilize a configuration service during bootstrapping) see [Standalone applications](https://docs.nestjs.com/standalone-applications).
```

**Comment by MickL** [comment]
Comitted your changes. I like them!

File: content/components.md [hunk 3]
```diff
@@ -206,14 +206,9 @@ This is how our directory structure should look now:
 
 #### Manual instantiation
 
-Thus far, we've discussed how Nest automatically handles most of the details of resolving dependencies.  In certain circumstances, you may need to step outside of the built-in Dependency Injection system and manually retrieve or instantiate providers.  We briefly discuss two such topics below.
+Thus far, we've discussed how Nest automatically handles most of the details of resolving dependencies.  In certain circumstances, you may need to step outside of the built-in Dependency Injection system and manually retrieve or instantiate providers. We briefly discuss two such topics below.
 
  To get existing instances, or instantiate providers dynamically, you can use [Module reference](https://docs.nestjs.com/fundamentals/module-ref).
  
  To get providers within the `bootstrap()` function (for example for standalone applications without controllers, or to utilize a configuration service during bootstrapping) see [Standalone applications](https://docs.nestjs.com/standalone-applications).
 To get existing or instantiate providers dynamically you can use [Module reference](https://docs.nestjs.com/fundamentals/module-ref).
```
Comment (johnbiundo) [inline content/components.md:214]: I think this sentence (last one) is left over from edits.  It's a duplicate of the sentence just above.

Comment (MickL) [reply content/components.md]: Fixed

**Comment by kamilmysliwiec** [comment]
Looks great! Thanks

**Pull request merged**